_posts/2020-06-05-Kim.md

```diff
@@ -6,6 +6,6 @@ date: 2020-06-05
 
 I'm looking forward to playing air hockey at Dave and Busters with my brother.
 
-![Air Hockey](https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.pinterest.com%2Fpin%2F75787206206008293%2F&psig=AOvVaw1-WEXAV0L3rP522aWp0fOZ&ust=1591452544607000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCJClwO_s6ukCFQAAAAAdAAAAABAE)
+![Air Hockey](https://media-cdn.tripadvisor.com/media/photo-s/06/6e/ef/c2/dave-and-buster-s.jpg)
 
 https://www.quotemaster.org/images/d2/d27bc480199b7f0f5414a268af8ed8d9.jpeg
```

Comment:
Nice! Who usually wins?
-Zach

Reply:
I'd say it's usually 6/10 times me, but don't ask my brother John ;)